WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListViewModel.swift

```diff
@@ -157,4 +208,50 @@ extension CustomFieldsListViewModel {
             self.fieldId = customField.id
         }
     }
+
+    struct PendingCustomFieldsChanges {
```

Comment:
I added this struct to have a centralized place for all the changes, this also made it easier to introduce the helper function `asJson` that we use for preparing the data for the saving.